numpy/core/fromnumeric.py

```diff
@@ -171,11 +171,11 @@ def reshape(a, newshape, order='C'):
     Notes
     -----
     It is not always possible to change the shape of an array without
-    copying the data. If you want an error to be raise if the data is copied,
+    copying the data. If you want an error to be raised if the data is copied,
     you should assign the new shape to the shape attribute of the array::
 
      >>> a = np.zeros((10, 2))
-     # A transpose make the array non-contiguous
+     # A transpose makes the array non-contiguous
```

Comment:
Hmm, that isn't right, probably goes way back to when there was contiguous and fcontiguos. A transpose changes C contiguous arrays into F contiguous arrays and vice versa.
```
In [1]: a = np.zeros((10, 2))

In [2]: a.flags
Out[2]: 
  C_CONTIGUOUS : True
  F_CONTIGUOUS : False
  OWNDATA : True
  WRITEABLE : True
  ALIGNED : True
  WRITEBACKIFCOPY : False
  UPDATEIFCOPY : False

In [3]: a.T.flags
Out[3]: 
  C_CONTIGUOUS : False
  F_CONTIGUOUS : True
  OWNDATA : False
  WRITEABLE : True
  ALIGNED : True
  WRITEBACKIFCOPY : False
  UPDATEIFCOPY : False
```

Reply:
Updated the line to reflect this. However, it might just be better to remove the explanation entirely, since this isn't information the user needs if they want to raise an error if data is copied when reshaping an array.

Reply:
I think this was correct before - the point is that the error message also says "contiguous", not `C-contiguous'. I suppose both could be updated, but that seems out of scope for a typo fix!

Revert this to just the typo fixes, and I'll merge it.

Reply:
Ah, got it. Reverted!